tests/Tests/ORM/Functional/Ticket/GH10808Test.php

```diff
@@ -1,101 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Doctrine\Tests\ORM\Functional\Ticket;
-
-use Doctrine\ORM\Mapping\Column;
-use Doctrine\ORM\Mapping\Entity;
-use Doctrine\ORM\Mapping\GeneratedValue;
-use Doctrine\ORM\Mapping\Id;
-use Doctrine\ORM\Mapping\JoinColumn;
-use Doctrine\ORM\Mapping\OneToOne;
-use Doctrine\ORM\Mapping\Table;
-use Doctrine\ORM\UnitOfWork;
-use Doctrine\Tests\OrmFunctionalTestCase;
-use PHPUnit\Framework\Attributes\Group;
-
-use function get_class;
-
-#[Group('GH10808')]
-class GH10808Test extends OrmFunctionalTestCase
```

Comment:
This test should be recovered / re-checked in detail: it is not about testing proxies, but deferred eager loading

Reply:
Needs to be done on 3.x then I guess.

Reply:
Done